assets/files/usr/local/bin/clusterinfo

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+APIURL=$(kubectl --kubeconfig /etc/kubernetes/kubeconfig config view -o jsonpath='{.clusters[0].cluster.server}')
```

Comment:
In case they decide to change the kubeconfig composition:
```
CURRENT_CONTEXT=$(kubectl --kubeconfig /etc/kubernetes/kubeconfig config view -o jsonpath='{.current-context}')
CLUSTER_NAME=$(kubectl --kubeconfig /etc/kubernetes/kubeconfig config view -o "jsonpath={.contexts[?(@.name == '""$CURRENT_CONTEXT""')].context.cluster}")
APIURL=$(kubectl --kubeconfig /etc/kubernetes/kubeconfig config view -o "jsonpath={.clusters[?(@.name == '""$CLUSTER_NAME""')].cluster.server}")
```

Reply:
Note that it already has the cluster_name right there.

Reply:
I'll update this, but it means calling kubectl 3 times, we may end up going back to less calls if the extra time becomes an issue.